src/main/resources/studentEmailFragment-courseJoin.html

```diff
@@ -18,4 +18,8 @@
     However, we recommend joining the course using your Google account,
     because it gives you more convenient access to all your feedback
     stored in TEAMMATES.
+</p>
+
+<p>
+    For any non-technical queries (e.g. assigned to the wrong team, misspelled name, request for deadline extension, etc.), you can contact your instructor(s) at: ${coOwnersEmails}
```

Comment:
Since the colon `:` is used within a sentence, you may want to add a period `.` after the list of emails.